packages/@okta/vuepress-site/docs/reference/api/groups/index.md

```diff
@@ -176,7 +177,8 @@ Enumerates Groups in your organization with pagination. A subset of Groups can b
 | after     | Specifies the pagination cursor for the next page of Groups                                | Query     | String   | FALSE    |         |
 | filter    | [Filter expression](/docs/reference/api-overview/#filtering) for Groups      | Query     | String   | FALSE    |         |
 | limit     | Specifies the number of Group results in a page                                            | Query     | Number   | FALSE    | 10000   |
-| q         | Searches the `name` property of Groups for matching value                                  | Query     | String   | FALSE    |         |
+| q         | Find groups that matches the `name` property                                               | Query     | String   | FALSE    |         |
+| search <ApiLifecycle access="ea" /> | Searches for groups with a supported [filtering](/docs/reference/api-overview/#filtering) expression for most properties                        | Query     | String   | FALSE    |         |
```

Comment:
"For most properties".  What properties would _not_ be included?  Do we have any more detail we could add to be a bit less vague?  E.g. if this is the same list that's enumerated below, maybe refer to it (or better yet, link to it)?

Reply:
I actually copied this description from Users search - https://developer.okta.com/docs/reference/api/users/#list-users. 

Looking at all group attributes (https://developer.okta.com/docs/reference/api/groups/#group-attributes), everything except for `_embedded`, `_links`, and `objectClass` is searchable. Would it be too specific if we put the description as `... for most [properties](#group-attributes)` except for `_embedded`, `_links`, and `objectClass``?

Reply:
I'll defer to the docs experts on this one.  There's a complex tradeoff here between 'we have to update the docs every time we change something' and 'people can't figure out what exactly our product does'.

Reply:
I'm leaning towards giving the people what they need to figure out what exactly our product does and dealing with updating the docs. Updating our docs is fairly easy (if Jiras are created for said changes in a timely manner).